packages/crawler/src/page-processors/page-processor-factory.ts

```diff
@@ -1,19 +1,36 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { Logger } from 'logger';
+import { reporterFactory } from 'accessibility-insights-report';
+import { inject, injectable } from 'inversify';
+import { GlobalLogger } from 'logger';
+import { AxePuppeteerFactory } from '../axe-puppeteer/axe-puppeteer-factory';
 import { CrawlerConfiguration } from '../crawler/crawler-configuration';
+import { AccessibilityScanOperation } from '../page-operations/accessibility-scan-operation';
+import { ClickElementOperation } from '../page-operations/click-element-operation';
+import { EnqueueActiveElementsOperation } from '../page-operations/enqueue-active-elements-operation';
+import { PageScanner } from '../scanner-operations/page-scanner';
+import { LocalBlobStore } from '../storage/local-blob-store';
+import { LocalDataStore } from '../storage/local-data-store';
 import { PageProcessorOptions } from '../types/run-options';
+import { ActiveElementsFinder } from '../utility/active-elements-finder';
 import { ClassicPageProcessor } from './classic-page-processor';
 import { PageProcessor } from './page-processor-base';
 import { SimulatorPageProcessor } from './simulator-page-processor';
 
+@injectable()
 export class PageProcessorFactory {
```

Comment:
This is good candidate to move to container as well

Reply:
it would've been good if there was no input parameter, I know still we can do it but it'll be more complicated